NGitLab/Models/ProjectUpdate.cs

```diff
@@ -111,4 +111,7 @@ public sealed class ProjectUpdate
 
     [JsonPropertyName("topics")]
     public List<string> Topics { get; set; }
+
+    [JsonPropertyName("ci_default_git_depth")]
+    public bool? CiDefaultGitDepth { get; set; }
```

Comment:
This property should be an `int?` instead. Refer to [the GitLab doc](https://docs.gitlab.com/api/projects/#edit-a-project).